docs/development.md

```diff
@@ -65,6 +65,24 @@ for package in packages/cmip_ref_core packages/cmip_ref packages/cmip_ref_metric
 done
 ```
 
+### Installing metric provider dependencies
+
+
+Some metric providers can use their own conda environments. The REF can manage
+these for you, provided that conda is installed.
+
+```bash
+ref --log-level=info providers create-env --provider ESMValTool
+```
+
+
+To update a conda-lock file, run for example:
+
+```bash
+uvx conda-lock -p linux-64 -f packages/ref-metrics-esmvaltool/src/cmip_ref_metrics_esmvaltool/requirements/environment.yml
```

Comment:
Is there a reason to only use a single platform? Or can we add extra platforms?

Testing this out on my MacBook results in an empty env with a warning:
`warning  libmamba Selected packages for category 'main' are empty. The lockfile might not be resolved for your platform (osx-arm64).`

Reply:
I'm assuming that it is because they don't have the appropriate dependencies available. For PMP osx-64 and linux-64 solve, but there is one dependency that doesn't have a distribution for win-64 or osx-arm64

Reply:
And esmvaltool solves for all but win-64.

Bad luck if you want to run the REF natively on Windows. We probably need to surface that information somewhere

Reply:
Indeed, ESMValTool doesn't work on Windows because not all dependencies are available for Windows. We usually recommend using WSL to people who are on Windows.

Reply:
Updated the command and lockfile to support osx in dadb44f